pkgs/data/icons/whitesur-icon-theme/default.nix

```diff
@@ -1,30 +1,19 @@
-{ lib
-, stdenvNoCC
-, fetchFromGitHub
-, gtk3
-, gnome
-, gnome-icon-theme
-, hicolor-icon-theme
-}:
+{ lib, stdenvNoCC, fetchFromGitHub, gtk3, hicolor-icon-theme }:
 
 stdenvNoCC.mkDerivation rec {
   pname = "Whitesur-icon-theme";
-  version = "2021-05-20";
+  version = "2021-08-26";
 
   src = fetchFromGitHub {
     owner = "vinceliuice";
     repo = pname;
     rev = version;
-    sha256 = "KboUYozTleOBKNun66g2oj7u/36hyQsPtRSk/x/LAWo=";
+    sha256 = "O7nb6X20HvnzldijP/fXqAs/2qE5JUg3DEMb84ZMQp4=";
   };
 
   nativeBuildInputs = [ gtk3 ];
 
-  buildInputs = [
-    gnome-icon-theme
-    gnome.adwaita-icon-theme
-    hicolor-icon-theme
-  ];
+  propagatedBuildInputs = [ hicolor-icon-theme ];
```

Comment:
curious, why does this need to be propagated?

Reply:
Re-reading the definition of `propagatedBuildInputs` it appears that I have misunderstood the meaning behind it and thus I shall replace it with the old `buildInputs`.